src/qt-solutions/qtpropertybrowser/src/qttreepropertybrowser.cpp

```diff
@@ -146,7 +146,7 @@ QtPropertyEditorView::QtPropertyEditorView(QWidget *parent) :
 
 void QtPropertyEditorView::drawRow(QPainter *painter, const QStyleOptionViewItem &option, const QModelIndex &index) const
 {
-    QStyleOptionViewItemV3 opt = option;
+    QStyleOptionViewItem opt = option;
```

Comment:
If you have time, it would be great to backport this to the official qt-solutions git repository !
    http://code.qt.io/cgit/qt-solutions/qt-solutions.git/
The qt-solutions should be a git-submodule in Mayo, I will do it in a further version ...